src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -754,6 +754,7 @@ class EthService(
               receivedInBlock.map(TransactionResponse(_, Some(block.header), pending = Some(false))))
           }
           .filter { case (sent, received) => sent.nonEmpty || received.nonEmpty }
+          .map { case (sent, received) => (sent.reverse, received.reverse) }
```

Comment:
Style/optional: I think it would more readable if it was inside the first `map`. Also would be great if that inner function returned something like `(a, b)`, not `(a.map(intoSomethingComplex), b.map(...))`

Reply:
I was thinking of replacing that `.filter.map` into sth like 
`          .collect {
            case (sent, received) if sent.nonEmpty || received.nonEmpty => (sent.reverse, received.reverse)
          }`
but i also consider it optional.

Reply:
Both comments addressed. I used collect and added intermediate vals in the map above.